build-info-extractor-npm/src/main/java/org/jfrog/build/extractor/npm/extractor/NpmBuildInfoExtractor.java

```diff
@@ -66,7 +68,8 @@ public Build extract(NpmProject npmProject) throws Exception {
         restoreNpmrc(workingDir);
 
         List<Dependency> dependencies = collectDependencies(workingDir);
-        return createBuild(dependencies);
+        String moduleId = org.apache.commons.lang.StringUtils.isNotBlank(module) ? module : npmPackageInfo.toString();
```

Comment:
Any reason why we use **org.apache.commons.lang.StringUtils** and not the imported **StringUtils**?